sourcecode/hub/config/services.php

```diff
@@ -31,4 +31,10 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'google' => [
+        'client_id' => env('GOOGLE_CLIENT_ID'),
+        'client_secret' => env('GOOGLE_CLIENT_SECRET'),
+        'redirect' => 'https://edlib-hub-stalin.ngrok.dev/google/callback/'
```

Comment:
The redirect URL here needs to be replaced with `/google/callback`, as per the remark about using relative paths in [Socialite's documentation](https://laravel.com/docs/10.x/socialite).